logaccess/spi/src/test/kotlin/LogFileServiceTest.kt

```diff
@@ -109,7 +109,7 @@ class LogFileServiceTest : WordSpec({
             logArchive.unpack(archiveContentDir)
 
             val expectedLogFiles = listOf("config.log", "analyzer.log", "reporter.log")
-            archiveContentDir.listFiles().orEmpty().map(File::getName) shouldContainExactlyInAnyOrder expectedLogFiles
+            archiveContentDir.walk().maxDepth(1).map(File::getName).shouldContainExactlyInAnyOrder(expectedLogFiles)
```

Comment:
Unfortunately, there seems to be no infix variant of this function for sequences.